ui/pages/desktop-pairing/index.scss

```diff
@@ -3,126 +3,39 @@
   flex-flow: column;
```

Comment:
I can't find this className anywhere is it being used? I don't think there is any jsx with just the `className="desktop-pairing"` If there is you should be able to use the `Box` component and box props for most of these styles

![Screenshot 2023-03-20 at 11 37 16 AM](https://user-images.githubusercontent.com/8112138/226435174-ef7e2de3-2797-4b54-9075-369be860992a.png)

Reply:
Thanks for pointing out, I think it's been used in the previous components, and could be removed.